src/packages/core/tests/server.test.ts

```diff
@@ -1230,58 +1261,287 @@ describe("server", () => {
       });
     });
 
-    it("responds with transfer-encoding: chunked responses when bufferification is triggered", async () => {
-      // this test needs to set BUFFERIFY_THRESHOLD before starting the server
-      await teardown();
+    describe("bufferification", () => {
+      it("responds over websockets when bufferification is triggered", async () => {
```

Comment:
This is an existing test and I didn't change anything; the git diff is just going a bit crazy here.